tests/sync/test_prior_auth_api.py

```
@@ -0,0 +1,337 @@
+        response = await communicator.receive_json_from()
+        self.assertEqual(response.get("status"), "generating")
+
+        # Wait for at least one proposal
+        received_proposal = False
+        for _ in range(5):  # Try up to 5 times
+            try:
+                response = await communicator.receive_json_from(timeout=10)
+                if "proposed_id" in response and "text" in response:
+                    received_proposal = True
+                    break
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)